crypto/err/openssl.txt

```diff
@@ -2207,12 +2208,17 @@ RSA_R_INVALID_SALT_LENGTH:150:invalid salt length
 RSA_R_INVALID_TRAILER:139:invalid trailer
 RSA_R_INVALID_X931_DIGEST:142:invalid x931 digest
 RSA_R_IQMP_NOT_INVERSE_OF_Q:126:iqmp not inverse of q
+RSA_R_KEY_PRIME_NUM_INVALID:165:key prime num invalid
 RSA_R_KEY_SIZE_TOO_SMALL:120:key size too small
 RSA_R_LAST_OCTET_INVALID:134:last octet invalid
 RSA_R_MGF1_DIGEST_NOT_ALLOWED:152:mgf1 digest not allowed
 RSA_R_MODULUS_TOO_LARGE:105:modulus too large
+RSA_R_MP_COEFFICIENT_NOT_INVERSE_OF_R:168:mp coefficient not inverse of r
+RSA_R_MP_EXPONENT_NOT_CONGRUENT_TO_D:169:mp exponent not congruent to d
+RSA_R_MP_R_NOT_PRIME:170:mp r not prime
 RSA_R_NO_PUBLIC_EXPONENT:140:no public exponent
 RSA_R_NULL_BEFORE_BLOCK_MISSING:113:null before block missing
+RSA_R_N_DOES_NOT_EQUAL_PRIMES:171:n does not equal primes
 RSA_R_N_DOES_NOT_EQUAL_P_Q:127:n does not equal p q
```

Comment:
These two are misleading messages. Second one should be "n does not equal p\*q" and first one should be "n does not equal product of prime factors" or something similar. It's not clear if second message should be fixed here, but it's no reason to not fix first one.

Reply:
Well, same can be said about more message. After all, these messages are rather for human consumption, and as such don't actually have to be macros without underscores...

Reply:
Very detailed description leads to longer marco name. But anyway I choose to fix a little this time...

Reply:
Detailed description has nothing to with macro name. Well, *initially* it's derived from macro name (by lowering and replacing underscores with spaces), but you can modify the message afterwards to your liking, or rather make it human readable.